Sources/Catchable.swift

```diff
@@ -149,6 +149,38 @@ public extension CatchMixin {
         return rp
     }
 
+    /**
+     The provided closure executes when this promise resolves, whether it rejects or not.
+     The chain waits on the returned `Guarantee`.
+
+         firstly {
+             setup()
+         }.done {
+             //…
+         }.ensure {
```

Comment:
this is a typo -- should be `ensureThen` after the rename

Reply:
Oops thanks.